regexes/device/mobiles.yml

```diff
@@ -14766,7 +14766,7 @@ LG:
       model: 'Stylus 2 Plus'
     - regex: 'LM-Q[67]10\.FG(?:[);/ ]|$)'
       model: 'Q7'
-    - regex: 'LM-(Q72[57][KLS]|Q610.FGN)(?:[);/ ]|$)'
+    - regex: 'LM-(Q72[57][KLS]|Q610\.FGN)(?:[);/ ]|$)'
```

Comment:
```suggestion
    - regex: 'LM-(Q72[57][KLS]|Q610.FGN)(?:[);/ ]|$)'
```

Reply:
The only failing tests I can see here is `LM-Q610(FGN))`, can we have this as a literal `\(` instead of a `.`?

Reply:
Oh I see, this also has to match `LM-Q610.FGN Build/O11019`

Reply:
4bf3e9e2